### DISCUSSION TITLE
Welcome to xy6020_wifi_ctlr Discussions!

### DISCUSSION THREAD

**Discussion by stealth20** [stealth20/xy6020_wifi_ctlr#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rudikx6** [comment]
Hi,

ich habe deine Steuerung mal aufgesetzt und es funktioniert soweit super.
Danke also erstmal für dieses coole Tool :)

Ich möchte die Smarte Steuerung des xy6020 gern für die Justage der Ladeleistung meiner Powerwall verwenden.
Netzteil -> xy6020 -> Victron-Laderegler -> Akku. 
Damit ich bei solarem Überschuss, entsprechend die Ladeleistung anpassen.

Deine zero feed-in Modus zielt auf die Entgegengesetzt Seite ab. richtig?
Akku -> xy6020 -> DC/AC Konverter -> Hausnetz mit Smartmeter


Bevor ich mir einen Branch deines Codes erstelle, würde ich gern wissen ob du eventuell auch diese Richtung schon bedacht hast oder etwas dazu eingebaut hast?

Gruß, Rudi

**Reply by stealth20** [reply]
Ja, bei mir sind es: Akku->xy6020->Mikrowechselrichter. Für Überschuss-Laden habe ich noch nichts vorgesehen. Dafür wäre der xy6020 mit 20A glaube ich zu schwach. Ist aber wahrscheinlich schnell implementiert. Viel Spass beim Programmieren.
